src/api/common/errors.js

```diff
@@ -58,6 +58,21 @@ function MissingLedgerHistoryError(message) {
 MissingLedgerHistoryError.prototype = new RippleError();
 MissingLedgerHistoryError.prototype.name = 'MissingLedgerHistoryError';
 
+function NonContiguousLedgerHistoryError(message) {
+  this.message = message ||
+    'Server has a gap in the ledger history in the specified range';
+}
+NonContiguousLedgerHistoryError.prototype = new RippleError();
+NonContiguousLedgerHistoryError.prototype.name =
+  'NonContiguousLedgerHistoryError';
+
+function PendingLedgerVersionError(message) {
+  this.message = message ||
+    'maxLedgerVersion is ahead of the server\'s ledger history';
```

Comment:
Might be more clear to say something like "maxLedgerVersion is greater than server's most recent validated ledger"